### PR TITLE
feat: init commit

### PR DESCRIPTION
**Pull request by a-y-ka**
Related to #2 

In this PR:
image

### CODE REVIEW

**Comment by zhumeisongsong** [comment]
You can add a simple description here:
image

**Comment by zhumeisongsong** [comment]
@a-y-ka 
Here are some issues of our training:
https://github.com/Atrae/front-end-developer-course/issues
Sorry I didn't info you at the beginning, the google home page task is  the  #2.

And I cancel the process confirm meeting in 4.5 and 4.7 since you have no issue that stucks you 🎉 
Congratulations that you finish it in a few days! So pround of you!

**Comment by a-y-ka** [comment]
Modified it!
スクリーンショット 2023-04-05 11 53 10

File: ayaka/html-google-home-page/css/style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+.my-icon {
+    background-color: darkmagenta;
+    border-radius: 50%;
+    padding: 5px 10px;
+}
+
+main {
+    width: 100%;
+    height: 600px;
```
Comment (zhumeisongsong) [inline ayaka/html-google-home-page/css/style.css:28]: Because the screen size are different. we usually height: 100vh instead of a static height.

**Comment by zhumeisongsong** [comment]
@a-y-ka 
About the layout:
1. The border color is not same, could you check it?
image

2. It seems the boder color of  button is different also.
image

3. Could you check the layout is broken in SP(mobile phone) display and share me the screenshot?

Thanks!

**Review by zhumeisongsong — APPROVED** [review]
Community Thumbs Up

**Pull request merged**